sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -202,6 +208,27 @@ class SparkSession private(
     new SparkSession(sparkContext, Some(sharedState))
   }
 
+  /**
+   * :: Experimental ::
+   * Create an identical copy of this `SparkSession`, sharing the underlying `SparkContext`
+   * and cached data. SessionState (SQL configurations, temporary tables, registered functions)
```

Comment:
You cannot refer to SessionState in this doc because its an internal thing. You can say "All the state of this session (i.e. configs, temporary views, registered functions)... "